src/__tests__/fireEvent.test.js

```diff
@@ -193,6 +193,46 @@ test('should not fire on disabled Pressable', () => {
   expect(handlePress).not.toHaveBeenCalled();
 });
 
+test('should not fire on disabled TextInput', () => {
+  const placeholder = 'Test placeholder';
+  const onChangeTextMock = jest.fn();
+  const NEW_TEXT = 'New text';
+
+  const { getByPlaceholderText } = render(
+    <View>
+      <TextInput
+        editable={false}
+        placeholder={placeholder}
+        onChangeText={onChangeTextMock}
+      />
+    </View>
+  );
+
+  fireEvent.changeText(getByPlaceholderText(placeholder), NEW_TEXT);
+  expect(onChangeTextMock).not.toHaveBeenCalled();
+});
+
+test('should not fire on disabled TextInput with nested Text', () => {
```

Comment:
There's no need for this test, can we remove it?

Reply:
Just to clarify, this test is also mentioned in https://github.com/callstack/react-native-testing-library/pull/517#discussion_r476390264.
Should we keep it therefore or remove it?